### PR TITLE
Profilemessage

### PR DESCRIPTION
**Pull request by Dhanashree-Patil95**
-depending on status of benificiary and progressbar value message to complete profile change

### CODE REVIEW

File: apps/front-end/src/pages/front-end/facilitator/Profile.js [hunk 1]
```diff
@@ -28,6 +30,26 @@ export default function Profile({ userTokenInfo, footerLinks }) {
   };
 
   const res = objProps(facilitator);
+  const percentage =
```
Comment (sagarkoshti1990) [inline apps/front-end/src/pages/front-end/facilitator/Profile.js:33]: Move to use effect hook and set in setProgress use state fuction.

File: lib/common-lib/src/components/layout/AppBar.js [hunk 2]
```diff
@@ -109,6 +111,34 @@ export default function AppBar({
 
   const res = objProps(facilitator)
 
+  const percentage = arrList(
```
Comment (sagarkoshti1990) [inline lib/common-lib/src/components/layout/AppBar.js:114]: same as above. move

**Review by sagarkoshti1990 — CHANGES_REQUESTED** [review]

File: apps/front-end/src/pages/front-end/facilitator/Profile.js (hunk 1 above)
Comment (Dhanashree-Patil95) [reply apps/front-end/src/pages/front-end/facilitator/Profile.js]: resolved

File: lib/common-lib/src/components/layout/AppBar.js (hunk 2 above)
Comment (Dhanashree-Patil95) [reply lib/common-lib/src/components/layout/AppBar.js]: resolved

**Review by sagarkoshti1990 — APPROVED** [review]

**Pull request merged**